src/modules/economy/views/MyEconomy.tsx

```diff
@@ -1,26 +1,30 @@
 import { useQuery } from '@apollo/client'
-import { SimpleGrid, Stack, Title } from '@mantine/core'
+import {
+  Card,
+  createStyles,
+  SimpleGrid,
+  Stack,
+  Text,
+  Title,
+} from '@mantine/core'
 import { Breadcrumbs } from 'components/Breadcrumbs'
 import { FullPageError } from 'components/FullPageComponents'
 import { FullContentLoader } from 'components/Loading'
-import {
-  AccountCard,
-  MyDeposits,
-  MyExpenditures,
-  MyPurchases,
-} from '../components'
+import { AccountCard, MyDeposits, MyExpenditures } from '../components'
 import { MY_BANK_ACCOUNT_QUERY } from '../queries'
 import { MyBankAccountReturns } from '../types.graphql'
+import { TransactionCard } from '../../dashboard/components/TransactionCard'
+import React from 'react'
 
-const breadCrumbItems = [
-  { label: 'Hjem', path: '/dashboard' },
-  { label: 'Min økonomi', path: '/economy/me' },
-]
-
-export const MyEconomy: React.VFC = () => {
+export const MyEconomy: React.FC = () => {
+  const { classes } = useStyles()
   const { data, loading, error } = useQuery<MyBankAccountReturns>(
     MY_BANK_ACCOUNT_QUERY
   )
+  const breadCrumbItems = [
+    { label: 'Hjem', path: '/dashboard' },
+    { label: 'Min økonomi', path: '/economy/me' },
+  ]
```

Comment:
We can move this outside the component since this never will change. Just place it right above